examples/IB/explicit/ex6/IBSimpleHierarchyIntegrator.cpp

```diff
@@ -170,28 +154,11 @@ IBSimpleHierarchyIntegrator::postprocessIntegrateHierarchy(const double current_
     IBHierarchyIntegrator::postprocessIntegrateHierarchy(
         current_time, new_time, skip_synchronize_new_state_data, num_cycles);
```

Comment:
This is in the wrong place - this class should postprocess its own data before calling the base class function. It's the opposite order as preprocessing (i.e., inner-outer-outer-inner).